code/api/src/modules/user/tests/userMutation.test.js

```diff
@@ -8,20 +8,23 @@ const request = require('supertest')(url);
 describe('User mutations', () => {
   afterEach(function() {
     request.post('/graphql')
-    .send({ query: 'mutation { userUpdate(id: 2, image: "https://en.pimg.jp/045/948/028/1/45948028.jpg") { id image } }'})
+    .send({ query: 'mutation { userUpdate(id: 2, image: "https://en.pimg.jp/045/948/028/1/45948028.jpg", description: "Tattooed seitan waistcoat austin asymmetrical chambray hot chicken man bun poke") { id image description } }'})
```

Comment:
This is an incredible description - please tell me you wrote that

Reply:
@priyapower I wish I could take credit. 🤣 https://hipsum.co/